mmv1/products/cgc/terraform.yaml

```diff
@@ -51,6 +51,23 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_mysql_instance_backup"
+        primary_resource_id: "instance"
+        vars:
+          mysql_instance_backup: "mysql-instance-backup"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_postgres_instance_backup"
+        primary_resource_id: "instance"
+        vars:
+          postgres_instance_backup: "postgres-instance-backup"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_sqlserver_instance_backup"
+        primary_resource_id: "default"
+        vars:
+          sqlserver_instance_backup: "sqlserver-instance-backup"
+        # ignore_read_extra:
+        #   - "root_password"
```

Comment:
This will probably need to be commented out to make the test work correctly

Reply:
I uncommented it so that now it's like line 31.